mne/_freesurfer.py

```diff
@@ -632,3 +748,43 @@ def _get_head_surface(surf, subject, subjects_dir, bem=None, verbose=None):
                 return _read_mri_surface(fname)
     raise IOError('No head surface found for subject '
                   f'{subject} after trying:\n' + '\n'.join(try_fnames))
+
+
+@verbose
+def _get_skull_surface(surf, subject, subjects_dir, bem=None, verbose=None):
```

Comment:
I suspect this is redundant with some BEM functions that will read all surfaces. You can just pick the skull from it.